package/PKGBUILD

```diff
@@ -5,7 +5,7 @@ pkgver=1.3
 pkgrel=1
 pkgdesc="Management utility to handle GPU switching for Optimus laptops (Git version)"
 arch=('any')
-url="https://github.com/Askannz/optimus-manager"
+url="https://github.com/ABHIRAMSHIBU/optimus-manager"
```

Comment:
huh?

Reply:
Holly crap.. sorry didnt mean to do that, just did that to test it.. Sorry